src/components/CustomModal.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Text, View, StyleSheet } from 'react-native';
+import Modal from 'react-native-modal';
+
+interface Props {
+  show: boolean;
+  children: React.ReactNode;
+}
+
+export const CustomModal = ({ show, children }: Props) => {
+  return (
+    <View style={styles.container}>
+      <Modal isVisible={show}>
+        <View style={styles.wrapper}>{children}</View>
+      </Modal>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: { flex: 1 },
+  wrapper: { flex: 1, justifyContent: 'center' },
+});
```

Comment:
According to designs, there should be a native alert https://zpl.io/VkAvqgZ (maybe without yellow buttons)

Reply:
have acccept for that